src/python_bindings/ind/bind_ind.cpp

```diff
@@ -0,0 +1,26 @@
+#include "bind_ind.h"
+
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+#include "algorithms/ind/faida/faida.h"
+#include "algorithms/ind/ind.h"
+#include "algorithms/ind/ind_algorithm.h"
+#include "algorithms/ind/spider/spider.h"
+#include "py_util/bind_primitive.h"
+
+namespace py = pybind11;
+
+namespace python_bindings {
+void BindInd(py::module_& main_module) {
+    using namespace model;
+
+    auto ind_module = main_module.def_submodule("ind");
+    py::class_<IND>(ind_module, "IND").def("__str__", &IND::ToString);
```

Comment:
`GetLhs` and `GetRhs` (and the ColumnCombination class, in the `data` module) should be bound too.